sup3r/models/multi_step.py

```diff
@@ -452,16 +355,14 @@ def generate(self, low_res, norm_in=True, un_norm_out=True,
         """
         logger.debug('Data input to the 1st step spatial-only '
                      'enhancement has shape {}'.format(low_res.shape))
-        s_exogenous = None
         t_exogenous = None
         if exogenous_data is not None:
-            s_exogenous = exogenous_data[:len(self.spatial_models)]
```

Comment:
maybe this is still worth keeping just for readability?

Reply:
Im trying to avoid the variable-to-variable pointer here: `s_exogenous = exogenous_data` which always seems kind of silly. Is it worth the readability? I could go either way.

Reply:
Oh yeah thats fair. I could go either way also. Could just pass exo_data[:len(self.spatial_models)] directly into generate.

Reply:
Yeah but the reason im not doing that now is so that the spatial models can have access to ALL of the exo_data including the 4km data that is technically assigned to the temporal model.

Reply:
Oh yeah I suppose that spatial models need up to `len(self.spatial_models) + 1` if theyre using the concat or add layers.